src/Symfony/Component/Form/Form.php

```diff
@@ -646,7 +646,9 @@ public function isEmpty(): bool
     public function isValid(): bool
     {
         if (!$this->submitted) {
-            throw new LogicException('Cannot check if an unsubmitted form is valid. Call Form::isSubmitted() before Form::isValid().');
+            throw new LogicException(
+                'Cannot check if an unsubmitted form is valid. Call Form::isSubmitted() and ensure that it\'s true before calling Form::isValid().'
+            );
```

Comment:
According to Coding standard, you have to put it all together in one line: https://fabbot.io/report/symfony/symfony/49667/f01a2e80d29f94fcdd854c881dffa81b6f8179ca

Reply:
Sorry for that, I'm fixing it ;)